apps/frontend/src/app/Components/Character/CharacterEditor/Content.tsx

```diff
@@ -175,6 +176,15 @@ function InTeam() {
   )
   if (!teamIds.length) return null
 
+  const onAddTeam = () => {
+    const teamId = database.teams.new()
```

Comment:
You can actually do this all in "one line", which eliminates one additional database write. But I am fine with this solution since its somewhat more verbose(and easier to maintain)
```
const teamId = database.teams.new({
  teamCharIds:[database.teamChars.new(characterKey)]
})
```